tools/ui-components/src/button.types.ts

```diff
@@ -3,8 +3,8 @@ type ButtonSize = 'small' | 'medium' | 'large';
 export interface ButtonProps {
   primary?: boolean;
   size?: ButtonSize;
-  backgroundColor?: string;
   label: string;
   customKey?: string;
   onClick: () => void;
+  theme?: 'light' | 'dark';
```

Comment:
Theme was added to accommodate the component wrapper in button story.

Reply:
Just a quick question - our database uses "default" and "night" for the theme settings on the user profile. Does this need to match those values?

Reply:
It should not be an issue since when we receive the user info, we can conditionally attach the relevant class to the html element.

Although it pays off to be consistent with naming all across, 'light'/'dark' seem to be the convention.